packages/opentelemetry-plugin-xml-http-request/test/xhr.test.ts

```diff
@@ -290,31 +290,35 @@ describe('xhr', () => {
             url,
             `attributes ${HttpAttribute.HTTP_URL} is wrong`
           );
+          assert.ok(
+            (attributes[keys[2]] as number) > 0,
+            'attributes transferSize <= 0'
```

Comment:
Oversight; I'll fix that when we figure out the right name.  Additionally, I find it awkward that these tests use numeric indices rather than just addressing the attributes by name.